src/azure-cli/azure/cli/command_modules/vm/tests/latest/test_vm_commands.py

```diff
@@ -5072,6 +5072,21 @@ def test_vm_ssh_key(self, resource_group):
         self.cmd('sshkey show -g {rg} -n k3')
 
 
+class VMInstallPatchesScenarioTest(ScenarioTest):
+    @ResourceGroupPreparer(name_prefix='cli_test_vm_install_patches_')
+    def test_vm_install_patches(self, resource_group):
```

Comment:
Add intentional wrong commands to test your validation code.

Reply:
Validation codes are removed and enum validness will be checked by Azure CLI framework .